proto/v1/vulnerability.proto

```diff
@@ -45,6 +45,8 @@ enum Severity {
 
 // Vulnerability provides metadata about a security vulnerability in a Note.
 message Vulnerability {
+  reserved 4;
```

Comment:
Please comment this out, but keep the wording.

Reply:
Are you saying to comment out the line "reserved 4"? The motive for this PR is to add this tag using the reserved keyword: https://developers.google.com/protocol-buffers/docs/proto#reserved.

Reply:
@tylerdatgoogle and I chatted about this and reached an agreement that we should revert this PR. Sorry for the troubles, @noahklein. My concern here is balancing good OSS stewardship and ensuring compatibility with internal Grafeas implementations. Making this an un-commented `reserved` field would conflict with internal usage and implications for other Grafeas users. Commenting out the `// reserved 4` has the same effect as the state before, so it's a no-op. I filed an issue for myself to look into this more comprehensively. Thank you both!